internal/ent/hooks/internalpolicy.go

```
@@ -0,0 +1,32 @@
+package hooks
+
+import (
+	"context"
+	"fmt"
+
+	"entgo.io/ent"
+
+	"github.com/theopenlane/core/internal/ent/generated"
+	"github.com/theopenlane/core/internal/ent/generated/hook"
+)
+
+// HookPolicySummarize summarizes the policy and produces a short human readable copy
+func HookPolicySummarize() ent.Hook {
+	return hook.On(func(next ent.Mutator) ent.Mutator {
+		return hook.InternalPolicyFunc(func(ctx context.Context, m *generated.InternalPolicyMutation) (generated.Value, error) {
+			details, ok := m.Details()
+			if !ok {
+				return nil, fmt.Errorf("details does not exists") // nolint:err113
+			}
```

I dont think we want to return an error, instead we want to only do the summary:

```if ok && details != ""`

---

Ha yes. I actually fixed this but didn't push. Will do that as soon as I get back on desk